content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -130,11 +127,8 @@ StorageClass has the field `allowVolumeExpansion` set to true.
 | Volume type          | Required Kubernetes version |
 | :------------------- | :-------------------------- |
 | gcePersistentDisk    | 1.11                        |
```

Comment:
Remove

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27